<launch>
    <arg name="base_frame" default="robot_0/base_link" />
	<arg name="odom_frame" default="robot_0/odom" />

    <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen">
		<remap from="scan" to="robot_0/base_scan_1"/>
		<remap from="map" to="gmap"/>

		<param name="odom_frame" value="$(arg odom_frame)"/>
		<param name="base_frame" value="$(arg base_frame)"/>
		<param name="map_frame" value="map"/>


		<param	name="delta" value="0.05"/>
	    <param name="xmin" value="-100"/>
	    <param name="ymin" value="-100"/>
	    <param name="xmax" value="100"/>
	    <param name="ymax" value="100"/>

		<param name="global_frame_id" value="map"/>
        <!--<rosparam command="load" file="$(find robot_describe)/config_nav/mapper_params.yaml" />-->
    </node>
	<node pkg="robot_describe" type="cost_map_node" name="cost_map_node" output="screen">
			<rosparam command="load" file="$(find robot_describe)/config_nav/my_costmap.yaml"/>
	</node>

</launch>